packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/hooks/useAuditLogsData.js

```diff
@@ -0,0 +1,49 @@
+import { useQueries } from 'react-query';
+import { useNotification, useFetchClient } from '@strapi/helper-plugin';
+import { useLocation } from 'react-router-dom';
+
+const useAuditLogsData = ({ canRead }) => {
+  const { get } = useFetchClient();
+  const { search } = useLocation();
+  const toggleNotification = useNotification();
+
+  const fetchAuditLogsPage = async ({ queryKey }) => {
+    const search = queryKey[1];
+    const { data } = await get(`/admin/audit-logs${search}`);
+
+    return data;
+  };
+
+  const fetchAllUsers = async () => {
+    const { data } = await get(`/admin/users`);
+
+    return data;
+  };
+
+  const queryOptions = {
+    enabled: canRead,
+    keepPreviousData: true,
+    retry: false,
+    staleTime: 1000 * 20,
+    onError() {
+      toggleNotification({
+        type: 'warning',
+        message: { id: 'notification.error', defaultMessage: 'An error occured' },
+      });
+    },
+  };
+
+  const [auditLogsData, userData] = useQueries([
```

Comment:
I think useQueries is not mandatory as we don't have a variable number of queries, we know there'll be 2. The code may be a little bit more straightforward with 2 useQuery calls